api/src/wfl/sink.clj

```diff
@@ -148,7 +148,7 @@
 
 (defn ^:private update-terra-workspace-sink
   [executor {:keys [fromOutputs workspace entityType identifier details] :as _sink}]
-  (when-let [{:keys [uuid outputs] :as workflow} (stage/peek-queue executor)]
+  (when-let [[_ {:keys [uuid outputs] :as workflow}] (stage/peek-queue executor)]
```

Comment:
Suggest `_type` (or is it `_name`) instead of just `_`?

Reply:
again, I don't agree that this adds clarity.

Reply:
I'm with Tom here, I find meaningful names to add clarity even if they're not currently being used.

Reply:
but it doesn't add any clarity because its not used - the whole point of `_` is you don't care what it is.